### PR TITLE
Set terminal.tab.activeBorder based on editor tab

### LINKED ISSUE #188: Light theme shouldn't set tab.activeBorder as white
`tab.activeBorder` is set to `#ffffff` which doesn't actually do anything on the tab:

Screen Shot 2021-07-18 at 4 13 46 PM

For the newly added style to terminal tabs though it looks a bit weird:

Screen Shot 2021-07-18 at 4 14 05 PM

Notice the white line on the left.

To clarify what this theme key does, here it is in `#ff0000`

Screen Shot 2021-07-18 at 4 14 28 PM
Screen Shot 2021-07-18 at 4 14 34 PM

### PR DESCRIPTION
**Pull request by Tyriar**
Fixes #188

### CODE REVIEW

File: src/classic/theme.js [hunk 1]
```diff
@@ -178,6 +178,7 @@ function getTheme({ style, name }) {
       "panelInput.border": pick({ light: primer.gray[2], dark: primer.gray[1] }),
 
       "terminal.foreground": primer.gray[6],
+      "terminal.tab.activeBorder": "#f9826c",
```
Comment (dominicegginton) [inline src/classic/theme.js:181]: I can not reproduce this change:

screenshot-Mon-19Jul21_01 15

I also can not find this attribute in the [VSCode Theme Color Documentation](https://code.visualstudio.com/api/references/theme-color). Is this a user setting?

Comment (Tyriar) [reply src/classic/theme.js:181]: This is in Insiders only right now, coming in the next version

Comment (dominicegginton) [reply src/classic/theme.js:181]: Awesome, thank you for your contribution. can you please provide a link to the documentation for the insiders build in this PR comments so we can reference it? 🚀

Comment (Tyriar) [reply src/classic/theme.js:181]: There's no documentation yet, here's where it's declared in code though:

https://github.com/microsoft/vscode/blob/7667e90bf82ece1395727c7d88a37114fe1488a4/src/vs/workbench/contrib/terminal/common/terminalColorRegistry.ts#L40-L44

**Review by dominicegginton — APPROVED** [review]

**Comment by dominicegginton** [comment]
🗒️ Just a note to the maintainer who merges this PR. This change affects the insiders build of VSCode. No effects of this change can be observed in stable release builds yet.

File: src/classic/theme.js (hunk 1 above)
Comment (simurai) [reply src/classic/theme.js:181]: > This is in Insiders only right now, coming in the next version

Thanks for clarifying. I think this should be fine to already merge. 👍

**Review by simurai — APPROVED** [review]

**Pull request merged**